### PR TITLE
QA fixes

### PR DESCRIPTION
**Pull request by DhaaraniCIT**

### CODE REVIEW

File: src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.html [hunk 1]
```diff
@@ -10,7 +10,7 @@
             <div class="tw-rounded-12-px tw-border-1-px tw-border-disabled-bg-color tw-mb-24-px ">
                 <div class="tw-p-24-px tw-border-b-1-px tw-border-b-separator tw-flex tw-items-start">
                     <div class="tw-pr-20-px tw-mt-4-px">
-                        <svg-icon-sprite src="people" width="20px" hight="20px" class="tw-text-mandatory-field-color"></svg-icon-sprite>
+                        <svg-icon-sprite src="threepeople" width="24px" hight="24px" class="tw-text-mandatory-field-color"></svg-icon-sprite>
```
Comment (ashwin1111) [inline src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.html:13]: use hyphen separators for names, please rename to multiple-users / multiple-people

File: src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.html [hunk 2]
```diff
@@ -40,11 +40,11 @@ <h5 class="!tw-text-faded-text-color tw-text-14-px tw-pt-8-px !tw-font-400 !tw-l
                 <app-email-multi-select-field
                     [form]="configurationForm"
                     [label]="'Send error notification to'"
-                    [isFieldMandatory]="true"
+                    [isFieldMandatory]="false"
```
Comment (ashwin1111) [inline src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.html:43]: I think this should be a mandatory field, without adding any option we cannot expect the user to hit save. As a result, the recipe will not be started. Please confirm this with Kamalini if we have to keep it non-mandatory. In the form group, you can keep this non-mandatory, but in the UI, I would like to see the `*` symbol

File: src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.ts [hunk 3]
```diff
@@ -46,11 +46,11 @@ export class GustoConfigurationComponent implements OnInit {
     const configurationPayload = GustoModel.constructGustoConfigurationPayload(this.configurationForm, this.org.id);
     this.gustoService.postConfigurations(configurationPayload).subscribe((response: GustoConfiguration) => {
       this.saveInProgress = false;
-      this.toastService.displayToastMessage(ToastSeverity.SUCCESS, 'Email address saved successfully');
+      this.toastService.displayToastMessage(ToastSeverity.SUCCESS, 'Your configurations have been saved successfully');
```
Comment (ashwin1111) [inline src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.ts:49]: Configuration saved successfully, confirm with Kamalini once

**Review by ashwin1111 — APPROVED** [review]

File: src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.html (hunk 2 above)
Comment (DhaaraniCIT) [reply src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.html:43]: @ashwin1111 email field need not be a mandatory field to save the configuration, got conformation from kamalini
https://fylein.slack.com/archives/C04GGNXSEAW/p1680501772113069?thread_ts=1680501385.295599&cid=C04GGNXSEAW

File: src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.ts (hunk 3 above)
Comment (DhaaraniCIT) [reply src/app/integrations/gusto/gusto-configuration/gusto-configuration.component.ts:49]: this content work is correct, got conformation from kamalini

**Pull request merged**